scss/4-elements/_forms.scss

```diff
@@ -64,23 +68,25 @@ textarea,
 .cads-textarea {
     @include cads-interactive-element();
 
+    width: 100%;
+    max-width: 50%;
```

Comment:
What do both together do when they're percentages?

Reply:
Ah...that should be width 50%, there's the width: 100% for the small breakpoint further down. Will change it.